webpack/components/Table/TableWrapper.js

```diff
@@ -53,7 +53,7 @@ const TableWrapper = ({
         <FlexItem>
           <Search patternfly4 {...{ onSearch, getAutoCompleteParams }} />
         </FlexItem>
-        <FlexItem breakpointMods={[{ modifier: FlexModifiers['align-right'] }]}>
+        <FlexItem align={{ default: 'alignRight' }}>
```

Comment:
oh I bet this is what's causing the Jenkins test failure on the other PR..

Reply:
yessir! this would be it